shipping-forecast.md

```diff
@@ -0,0 +1,2 @@
+Viking, North Uitsere, South Uitsere: Low, 920, Falling quickly. Good, becoming poor. Heavy icing expected later.
```

Comment:
A bit too much icing for my liking, but I guess it's getting near to Christmas...
```suggestion
Viking, North Uitsere, South Uitsere: Low, 920, Falling quickly. Good, becoming poor. Icing expected later.
```